lib/util/hud_lists.rb

```diff
@@ -428,6 +428,10 @@ def relationship_to_ho_h_map
     }.freeze
   end
 
```

Comment:
This file is generated, so it shouldn't be edited directly. Maybe you can just put these directly on the referral models where they are used? We don't really use enums elsewhere throughout the warehouse or HMIS. I would want to discuss with Elliot before introducing this as a common pattern.

Reply:
Ah oops, I should have checked the header! 🤦 

I moved it into HudUtility since I want to share the key transform method